pyproject.toml

```
@@ -46,11 +46,11 @@ test = [
 ]
 mace = ["mace-torch>=0.3.11"]
 mattersim = ["mattersim>=0.1.2"]
-
 orb = [
     "orb-models@git+https://github.com/orbital-materials/orb-models#egg=637a98d49cfb494e2491a457d9bbd28311fecf21",
 ]
 sevenn = ["sevenn>=0.11.0"]
+metatensor = ["metatrain>=2025.4"]
```

```suggestion
metatensor = ["metatensor-torch >=0.7,<0.8", "metatrain>=2025.4"]
```

This way we can more easily update the code when the metatomic refactor is done, and the two dependencies of this code are explicit (instead of implicitly pulling metatensor-torch as a dependency of metatrain). Without the version constrain I could see a world where metatrain updates to new versions of metatensor-torch, but this code does not support it.